### ISSUE TITLE
Directly support docusaurus

### ISSUE THREAD

**Issue by fredemmott** [hhvm/hh-apidoc#29]
This will basically be the same as markdown output, but pages should include a header: https://docusaurus.io/docs/en/doc-markdown

**Comment by fredemmott** [comment]
cc @JoelMarcey , @yangshun :)

**Comment by JoelMarcey** [comment]
Also cc: @endiliey

**Comment by JoelMarcey** [comment]
Just so I can context switch back on this for a sec -- this is having hhapi-doc support outputting a format consumable by Docusaurus?

**Comment by fredemmott** [comment]
Yep - basically the same as markdown mode, but with the metadata block added
